lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,27 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # Car behaviour
+  class Car
+    attr_reader :color, :model, :year, :current_speed
+
+    def initialize(car)
+      @year = car[:year] || 2015
```

Comment:
then you can use `.fetch` for hash
`@year = car_attributes.fetch(:year, 2015)`
http://ruby-doc.org/core-2.4.1/Hash.html#method-i-fetch